### DISCUSSION TITLE
ESP32 + CP2102 läuft nicht mit esp32s3.factory.bin

### DISCUSSION THREAD

**Discussion by huste511** [tbnobody/OpenDTU#1301] [Q&A]
Gekauft wurden diese ESP32: https://www.aliexpress.com/item/1005004887573569.html in der Variante „CP2102 TYPE-C“. 

CP2102 ist meinem Verständnis eine usb-uart-bridge. Der ESP32 wird auch entsprechend mit Treiber am Rechner erkannt.

Auf https://github.com/tbnobody/OpenDTU/wiki/Binary-Installation lese ich heraus, dass es ich eine opendtu-generic_esp32s3.factory.bin flaschen müßte. Funktioniert leider nicht. Es wird kein WiFi aufgezogen, dass man loslegen könnte.

Näch längeren Tests war auch opendtu-generic_esp32s3_usb.factory.bin erfolglos. Erst opendtu-generic.factory.bin oder opendtu-generic_esp32.factory.bin laufen und öffnen ein Wifi, mit dem man weiter arbeiten kann.

Ich habs jetzt am laufen, würde aber gerne wissen, warum genau es nicht lief bzw. was wie man beim nächsten mal mit anderem ESP32 das richtige auswählt.

thx

**Comment by tbnobody** [comment]
> Modul modell: ESP32-WROOM-32

Das ist ein ESP32 und kein ESP32S3

**Comment by huste511** [comment]
Ahhhh ... thx ... für mich war ESP immer gleich ESP. Das es diese noch mit anderen CPUs gibt, hatte ich nicht auf dem Plan.
Hatte mich dann am Comment orientiert und mich da dann mit googeln CP2102 = uart auf den falschen Weg locken lassen. ;-)
